apps/web/modules/bookings/views/bookings-single-view.tsx

```diff
@@ -149,6 +148,7 @@ export default function Success(props: PageProps) {
     props?.userTimeFormat ? props.userTimeFormat === 24 : isBrowserLocale24h()
   );
   const { data: session } = useSession();
+  const isHost = bookingInfo?.user?.id === session?.user?.id;
```

Comment:
In case of collective team event type, there could be more than one host and they are stored in booking.Hosts

Reply:
We used this logic down below on L:320 - does this mean that is wrong also\?